test/models/entity_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```

Comment:
- [ ] Add tests for the various models.

Reply:
- I have written all tests in `spec` folder, not in `test` folder, and I also have mentioned testing command in my readme that in order to run the tests, you can run in terminal `rspec spec`.